### PR TITLE
Update the autoSkim.py for the Run3 skims

### PR DESCRIPTION
**Pull request by sam7k9621**
#### PR description:
This PR is to update the autoSkim for the Run3 skims.
Following the discussion in #37656, the autoSkim is now for Run3.

#### PR validation:
None

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
backport of #37655

### CODE REVIEW

**Comment by qliphy** [comment]
please test

**Comment by jordan-martins** [comment]
+pdmv

**Comment by qliphy** [comment]
+1

File: Configuration/Skimming/python/autoSkim.py [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
- 'SingleMuon' : 'MuonPOGSkim+ZMu+MuTau+LogError+LogErrorMonitor',
+ 'SingleMuon' : 'ZMu+LogError+LogErrorMonitor',
  'DoubleMuon' : 'LogError+LogErrorMonitor',
  'MuonEG' : 'TopMuEG+LogError+LogErrorMonitor',
- 'DoubleMuonLowMass' : 'BPHSkim+LogError+LogErrorMonitor',
- 'MuOnia' : 'BPHSkim+LogError+LogErrorMonitor',
- 'Charmonium' : 'MuonPOGJPsiSkim+BPHSkim+LogError+LogErrorMonitor',
+ 'DoubleMuonLowMass' : 'LogError+LogErrorMonitor',
+ 'MuOnia' : 'LogError+LogErrorMonitor',
+ 'Charmonium' : 'MuonPOGJPsiSkim+LogError+LogErrorMonitor',
```
Comment (tvami) [inline Configuration/Skimming/python/autoSkim.py:19]: These dont exist anymore

File: Configuration/Skimming/python/autoSkim.py [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
  'JetHT' : 'JetHTJetPlusHOFilter+LogError+LogErrorMonitor',
- 'DisplacedJet' : 'LogError+LogErrorMonitor',
- 'MET' : 'HighMET+LogError+LogErrorMonitor',
+ 'DisplacedJet' : 'EXODisplacedJet+EXODelayedJet+EXODTCluster+EXOCSCCluster+LogError+LogErrorMonitor',
+ 'MET' : 'EXOHighMET+EXODelayedJetMET+LogError+LogErrorMonitor',
  'SingleElectron' : 'LogError+LogErrorMonitor', #to be updated if we will have EGamma as Run-2 (2018), or splitting as 2016,2017
- 'SinglePhoton' : 'SinglePhotonJetPlusHOFilter+EXOMONOPOLE+LogError+LogErrorMonitor', #to be updated if we will have EGamma as Run-2 (2018), or splitting as 2016,2017
- 'DoubleEG' : 'ZElectron+LogError+LogErrorMonitor', #to be updated if we will have EGamma as Run-2 (2018), or splitting as 2016,2017
- 'EGamma':'SinglePhotonJetPlusHOFilter+ZElectron+EXOMONOPOLE+LogError+LogErrorMonitor',
+ 'SinglePhoton' : 'EXOMONOPOLE+LogError+LogErrorMonitor', #to be updated if we will have EGamma as Run-2 (2018), or splitting as 2016,2017
```
Comment (tvami) [inline Configuration/Skimming/python/autoSkim.py:10]: Same thing here

**Comment by tvami** [comment]
Also what's the backport of this that entered 12_4_3 ?

**Comment by sam7k9621** [comment]
> Also what's the backport of this that entered 12_4_3 ?

There isn't one, since it was just merged into the central CMSSW (12_5_X).
Do we need a backport to 12_4_3?
The current backport refers to 12_4_X.

**Comment by tvami** [comment]
> The current backport refers to 12_4_X.

Which is that backport? This PR is towards 12_5_X, I'm asking which is the version of this PR that's towards 12_4_X. It's certainly not https://github.com/cms-sw/cmssw/pull/37655 , right?

**Comment by sam7k9621** [comment]
Hi @kskovpen  and @tvami ,
I am now preparing the backport for 12_4_X and also a new PR to address the comment (removing the unused PD).

**Pull request merged**